### PR TITLE
Updated ES translation

### PR DESCRIPTION
**Pull request by RAVMN**

### CODE REVIEW

File: assets/_locales/es/messages.json [hunk 1]
```diff
@@ -731,6 +731,10 @@
     "message": "Últimos enlaces de página usados",
     "description": "Last used page links"
   },
+  "settings_anilist_update_ui": {
+    "message": "Actualizar UI",
```
Comment (lolamtisch) [inline assets/_locales/es/messages.json:735]: The option adds a section to directly update an entry. I am not sure but something in the direction "UI de/para editar" is probably better here. Or what do you think?
image

**Review by lolamtisch — DISMISSED** [review]

File: assets/_locales/es/messages.json (hunk 1 above)
Comment (RAVMN) [reply assets/_locales/es/messages.json:735]: What do you think about "cuadro de actualización"(update box) or "bloque de actualización" (update block) ?
As it is not usual for final users to speak about "UI" in spanish

Comment (RAVMN) [reply assets/_locales/es/messages.json:735]: What do you think about "cuadro de actualización"(update box) or "bloque de actualización" (update block) ?
As it is not usual for final users to speak about "UI" in spanish

Comment (lolamtisch) [reply assets/_locales/es/messages.json:735]: `bloque de actualización` sounds fine to me. Is `cuadro` really used as a term for a UI box? If yes then I am fine with both.

**Review by henrik9999 — APPROVED** [review]

**Pull request merged**